tracks/julia/exercises/secret-handshake/mentoring.md

```diff
@@ -8,6 +8,23 @@
 ### Example solution
 
 ````
+pdb5627's solution (slightly edited)
+
+This short and simple solution is fast, extremely easy to read, and clearly demonstrates the bitwise operations that most solutions use.
+Four of the lines are obviously very similar to each other and it's totally fine to write code that is "unrolled" like this.
+
+```julia
+function secret_handshake(n)
+    s = []
```

Comment:
```suggestion
    s = String[]
```

Reply:
I deliberately left this off, but I suppose it's not so scary.